presto-main/src/main/java/io/prestosql/server/protocol/Slug.java

```diff
@@ -30,6 +30,11 @@
 
     private static final SecureRandom RANDOM = new SecureRandom();
 
+    /**
+     * Instantiates a new Slug object.
```

Comment:
I think it would be nice to explain in a description what this Slug object represents